server/index.js

```diff
@@ -28,38 +21,6 @@ function register (server, options, next) {
   // mapreduce is required for `db.query()`
   PouchDB.plugin(require('pouchdb-mapreduce'))
```

Comment:
I find it! the problem is about this issue .

https://github.com/pouchdb/pouchdb/issues/5567